srcpkgs/vmpk/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'vmpk'
+pkgname=vmpk
+version=0.7.1
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config libxslt docbook-xsl"
+makedepends="qt5-x11extras-devel qt5-tools-devel drumstick-devel"
+short_desc="MIDI events generator and receiver"
+maintainer="Mohammad Amin Sameti <mamins1376@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://vmpk.sourceforge.net"
+distfiles="${SOURCEFORGE_SITE}/vmpk/${version}/vmpk-${version}.tar.bz2"
+checksum=06d78872b8c23976cd1ed4e66ac6a3bfce93af28ce8db96c2c015cf185ce651b
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-qmake qt5-host-tools"
+fi
+
+post_install() {
+	for doc in TODO NEWS ChangeLog README AUTHORS; do
```

Comment:
I don't think we ship those files.
We may ship NEWS, but I'm not sure.

Reply:
`TODO` is shipped in almost none, `vdoc README` in at least 479 packages, `NEWS` in 37 and `AUTHORS` in 47, all including this one.

I assume `TODO` should not be shipped for sure. What about others?